tests/foreman/sys/test_pulp3_filesystem.py

```diff
@@ -0,0 +1,74 @@
+"""Tests for Pulp3 file system.
+
+:Requirement: Pulp3 installed
+
+:CaseAutomation: Automated
+
+:CaseLevel: System
+
+:CaseComponent: Pulp
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+
+"""
+import pytest
+
+from robottelo.ssh import get_connection
+
+
+@pytest.mark.upgrade
+def test_selinux_status():
+    """Test SELinux status.
+
+    :id: 43218070-ac5e-4679-b74a-3e2bcb497a0a
+
+    :expectedresults: SELinux is enabled and there are no denials
+
+    """
+    with get_connection(timeout=100) as connection:
+        # check SELinux is enabled
+        result = connection.run('getenforce')
+        assert 'Enforcing' in result.stdout[0]
+        # check there are no SELinux denials
+        result = connection.run('ausearch --input-logs -m avc -ts today --raw')
+        assert result.return_code == 1
+
+
+@pytest.mark.upgrade
+def test_file_layout_and_settings():
+    """Test directories and systemd settings for Pulp3 file system layout.
+
+    :id: e584faea-dede-4895-8d7f-411cb074f6c0
+
+    :expectedresults: systemd files are present, Pulp3 directories are present.
+
+    """
+    with get_connection(timeout=100) as connection:
+        # check pulpcore settings
+        result = connection.run('systemctl cat pulpcore-content.socket')
+        assert result.return_code == 0
+        assert 'ListenStream=/run/pulpcore-content.sock' in result.stdout[3]
+        result = connection.run('systemctl cat pulpcore-content.service')
+        assert result.return_code == 0
+        assert 'Requires=pulpcore-content.socket' in result.stdout[2]
+        result = connection.run('systemctl cat pulpcore-api.service')
+        assert result.return_code == 0
+        assert 'Requires=pulpcore-api.socket' in result.stdout[3]
+        # check pulp3 configuration file present
+        result = connection.run('test -f /etc/pulp/settings.py')
+        assert result.return_code == 0
+        # check pulp3 directories present
+        directories = [
+            '/var/lib/pulp',
+            '/var/lib/pulp/assets',
+            '/var/lib/pulp/media',
+            '/var/lib/pulp/tmp',
+            '/var/lib/pulp/static',
+        ]
+        for directory in directories:
+            msg = directory + " must exist"
+            assert connection.run('test -d ' + directory).return_code == 0, msg
```

Comment:
Can you use f-strings for the above two lines?

Reply:
OK, done